### PR TITLE
update scd4x: expose temp offset & altitude config + docs entry update

### PR DESCRIPTION
**Pull request by mbz4**
update scd4x: expose temp offset & altitude config + docs entry update
docs:
image

image
header:
image
cpp:
image
image
ignore the ens16x test edit (WIP...) .-.

### CODE REVIEW

**Comment by mbz4** [comment]
tested working w/ nodered & comparing vs other temperature sensors (dallas, dht22, open meteo current forecast) 
image

File: doc/node_help/scd4x.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@ Sensirion SCD4X CO2 sensor
 
 ..  code-block:: cpp
 
-    scd4x(name)[.i2c(sda,scl)][.with_address(i2c_address)][.with_filter(filter)];
+    scd4x(name[, temp_comp][, altitude])[.i2c(sda,scl)][.with_address(i2c_address)][.with_filter(filter)];
```
Comment (ulno) [inline doc/node_help/scd4x.rst:6]: it should be [, temp_comp[, altitude]] as you cannot specify only the altitude (without specifying the temperature - cpp restriction - in python that works).

File: doc/node_help/scd4x.rst [hunk 2]
```diff
@@ -49,6 +49,21 @@ Parameters
...
+    - Recommend determining offset when installed and running under typical operational conditions and in thermal equilibrium.
+    - Inside the code it can be addressed via ``IN(temp_comp)``.
+
+
+- ``altitude`` (optional; default = 57)
+    - Typically, the sensor altitude is set once after device installation. 
+    - The default sensor altitude value is set to 0 meters above sea level. 
+    - Valid input values are between 0 to 3000 m. 
+    - Inside the code it can be addressed via ``IN(altitude)``.
+
```
Comment (ulno) [inline doc/node_help/scd4x.rst:65]: :+1:

File: lib/node_types/esp/src/dev_ens16x.cpp [hunk 3]
```diff
@@ -1,5 +1,5 @@
 // dev_ens16x.cpp
-
+//WIP
```
Comment (ulno) [inline lib/node_types/esp/src/dev_ens16x.cpp:2]: important remark, but let's stick to the common notation and add a TODO: before

**Review by ulno — COMMENTED** [review]
thanks, just minor changes and it's ready to merge

File: doc/node_help/scd4x.rst
Comment (mbz4) [inline doc/node_help/scd4x.rst:1]: fixed [, temp_comp[, altitude]]

File: lib/node_types/esp/src/dev_ens16x.cpp (hunk 3 above)
Comment (mbz4) [reply lib/node_types/esp/src/dev_ens16x.cpp]: done

File: doc/node_help/scd4x.rst (hunk 1 above)
Comment (mbz4) [reply doc/node_help/scd4x.rst]: done

**Pull request merged**